### PR TITLE
common/TrackedOP: make slow_op_threshold work on ms or us.

### PR DESCRIPTION
**Pull request by majianpeng**
For all-nvme-ssd Ceph cluster, dupration of Op is ms or us. So we
need make slow_op_threshold in this range.

Signed-off-by: Jianpeng Ma <jianpeng.ma@intel.com>

- [ ] References tracker ticket
- [ ] Updates documentation if necessary
- [ ] Includes tests for new functionality or reproducer for bug

### CODE REVIEW

File: src/common/TrackedOp.h [hunk 1]
```diff
@@ -62,7 +62,7 @@ class OpHistory {
   uint32_t history_size;
   uint32_t history_duration;
   uint32_t history_slow_op_size;
-  uint32_t history_slow_op_threshold;
+  float history_slow_op_threshold;
```
Comment (batrick) [inline src/common/TrackedOp.h]: better would be `double`

File: src/common/options.cc [hunk 2]
```diff
@@ -1238,10 +1238,10 @@ std::vector<Option> get_global_options() {
     .add_service("mon")
     .set_description("max number of slow historical MON OPS to keep"),
 
-    Option("mon_op_history_slow_op_threshold", Option::TYPE_SECS, Option::LEVEL_ADVANCED)
-    .set_default(10)
+    Option("mon_op_history_slow_op_threshold", Option::TYPE_FLOAT, Option::LEVEL_ADVANCED)
+    .set_default(10.0)
     .add_service("mon")
-    .set_description("duration of an op to be considered as a historical slow op"),
+    .set_description("duration of an op to be considered as a historical slow op(seconds)"),
```
Comment (batrick) [inline src/common/options.cc]: `duration (seconds) of an op` would be better

**Comment by majianpeng** [comment]
@batrick  update by your suggestion. please review.

**Comment by tchaikov** [comment]
@majianpeng needs rebase

**Comment by majianpeng** [comment]
@tchaikov .  Conflicted because of commit: f21b08f9b91d84d518de205 which common/TrackedOp: make settings atomic,  To avoid locks on configuration changes. but only c++20 support atomic_t<double/float>.  I think only change the unit from second to ms or us.  How about your options?

**Comment by majianpeng** [comment]
because  https://github.com/ceph/ceph/commit/f21b08f9b91d84d518de205fd81824b49b91796f,  i change unit from second to millisecond. @batrick  please review.

**Comment by majianpeng** [comment]
retest this please

**Comment by majianpeng** [comment]
@batrick @tchaikov . please review . Thanks!

**Comment by majianpeng** [comment]
retest this please

**Pull request closed without merging**